src/packages/core/notification/layouts/default/notification-layout-default.element.ts

```diff
@@ -16,11 +24,46 @@ export class UmbNotificationLayoutDefaultElement extends LitElement {
 		return html`
 			<uui-toast-notification-layout id="layout" headline="${ifDefined(this.data.headline)}" class="uui-text">
 				<div id="message">${this.data.message}</div>
+				${this.#renderStructuredList(this.data.structuredList)}
 			</uui-toast-notification-layout>
 		`;
 	}
 
-	static override styles = [UmbTextStyles];
+	#renderStructuredList(list: unknown) {
+		if (!this.data.structuredList) return nothing;
+		if (typeof list !== 'object' || list === null) return nothing;
+
+		return html`${Object.entries(list).map(
+			([property, errors]) =>
+				html`<div class="structured-list">
+					<span>${property}:</span>
+					<ul>
+						${this.#renderListItem(errors)}
+					</ul>
+				</div>`,
+		)}`;
+	}
+
+	#renderListItem(items: unknown) {
+		if (Array.isArray(items)) {
+			return items.map((item) => html`<li>${item}</li>`);
+		} else {
+			return html`<li>${items}</li>`;
+		}
+	}
+
+	static override styles = [
+		UmbTextStyles,
+		css`
+			.structured-list ul {
+				margin: 0;
+			}
+			.structured-list span {
+				display: block;
+				padding: var(--uui-size-3) 0 var(--uui-size-1);
+			}
```

Comment:
I would think that a `<p>` element gives approx. the same styling without having to force any display:block or padding. Have you tried that?

Reply:
`<p>` gives a lot of margin, about 24px top and bottom. This is all set with the `.uui-text` class. But at least wouldn't have to force display:block.
Originally I had `h2` on, thought it would make more sense HTML scematic wise, but that requires  even more styling due to our class.